src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -118,6 +124,30 @@ def _list_devices(self, cmd):
 
         return 0, result, ""
 
+    def _list_services(self, cmd):
+        svc_type = None
+        if 'svc_type' in cmd:
+            svc_type = cmd['svc_type']
+
+        completion = self._oremote("list_services", svc_type)
+
+        self._wait([completion])
+
+        services = completion.result
+
+        if len(services) == 0:
+            return 0, "", "No services reported"
+        else:
+            lines = []
+            for s in services:
```

Comment:
While formatting for display, might want to do some sorting (perhaps sort by node, then type, then name)

Reply:
Done. I sorted them in slightly different order, but I think it makes sense.